cts/cts-cli.in

```diff
@@ -1903,7 +1899,7 @@ function test_tools() {
 
     desc="Reset shadow instance (mismatch)"
     cmd="crm_shadow --reset $shadow --batch"
-    test_assert $CRM_EX_USAGE 0
+    test_assert $CRM_EX_OK 0
```

Comment:
On second thought, you could make a good case for still requiring `--force` for an instance mismatch in reset. Since it typically uses an already existing shadow file, you could assume that the user wants to reset the currently active one.

Reply:
Yeah I think it does make sense to keep --force for reset, just to make sure they have to think about which one they want